pandas/core/indexes/datetimes.py

```diff
@@ -342,13 +344,17 @@ def dtype(self):
         return self._eadata.dtype
 
     @property
-    def _values(self):
-        # tz-naive -> ndarray
-        # tz-aware -> DatetimeIndex
-        if self.tz is not None:
-            return self._eadata
-        else:
-            return self.values
+    def tz(self):
+        # GH 18595
+        return self._eadata.tz
+
+    @tz.setter
+    def tz(self, value):
+        # GH 3746: Prevent localizing or converting the index by setting tz
+        raise AttributeError("Cannot directly set timezone. Use tz_localize() "
+                             "or tz_convert() as appropriate")
+
+    tzinfo = tz
```

Comment:
i have a mild preference for these being down in the Wrapping DatetimeArray section; short-term the only real difference is diff noise